README.md

```
@@ -245,13 +267,124 @@ let walletMetadata = try WalletMetadata(presentationDefinitionURISupported: true
+
+```swift
+    let authorizationRequest : AuthorizationRequest = try openID4VP.authenticateVerifier(authorizationRequest: [String: Any], trustedVerifiers: [Verifier], shouldValidateClient: Bool)
+```
+
+#### Parameters
+
+| Name                           | Type            | Required | Default Value | Description                                                                                                                                                                                                                                                                             |
+|--------------------------------|-----------------|----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| urlEncodedAuthorizationRequest | String          | Yes      | N/A           | URL encoded query parameter string containing the Verifier's authorization request                                                                                                                                                                                                      |
+| authorizationRequest           | [String : Any]  | Yes      | N/A           | authorization request                                                                                                                                                                                                                                                                   |
+| trustedVerifiers               | [Verifier]      | Yes      | N/A           | A list of trusted Verifier objects each containing a clientId, responseUri, jwksUri and allowUnsignedRequest which is used to verify if the Authorization Request if from known Verifier (refer [here](#verifier-parameters) for more details)                                          |
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)